devnets/specs/centauri.nix

```diff
@@ -107,8 +107,7 @@
               execCommands = [ "relay" "--config" configPathContainer ];
               inherit configPathSource configPathContainer pkgs packages
                 devnetTools;
-              dependsOn = dependsOnCreateConnection;
-              # safely restart on failure due to connectivity loss for instance
+              dependsOn = dependsOnCreateClient;
```

Comment:
In this case, `hyperspace-relay` will use `config.toml` without an update after `hyperspace-create-connection`.

Reply:
Well caught

Reply:
i do not see any errors in logs.

Reply:
the order should be
1. create client
2. create connection
3. create channels
4. relay